<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerDetailWidget</class>
 <widget class="QWidget" name="WorkerDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>工单详情</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FinanMemberWidget{background-color: rgb(255, 255, 255);border-radius:8px;}</string>
  </property>
  <widget class="QWidget" name="TopWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>52</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>840</width>
     <height>52</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 252, 252);
border-bottom:1px solid rgb(222, 222, 222);
border-top-right-radius:8px;
border-top-left-radius:8px;</string>
   </property>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>15</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>18</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
    </property>
    <property name="text">
     <string>工单详情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>32</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>28</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/EntCenterWidget/image/close_sel.png);};</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelsubmitcolor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:10px solid rgba(0,0,255,1);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelEndcolor">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:10px solid rgba(0,0,255,1);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartcolor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:10px solid rgba(0,0,255,1);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgressStart">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>68</y>
     <width>170</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgba(0,0,240,1)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgressEnd">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>68</y>
     <width>170</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgba(0,0,240,1)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>85</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>提交工单</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>85</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>处理中</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>85</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>已完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeCreate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>105</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>2020-10-01 12：00：00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeStart">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>105</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>2020-10-01 12：00：00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeEnd">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>105</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>2020-10-01 12：00：00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodeName">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>120</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>18</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textContent">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>145</y>
     <width>600</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 224, 0%);   border: 0px solid black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>300</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>18</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>附件</string>
   </property>
  </widget>
  <widget class="QListView" name="listFiles">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>325</y>
     <width>600</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 224, 0%);   border: 0px solid black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOkay">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
